### ISSUE TITLE
Markbot Validation

### ISSUE THREAD

**Issue by MJholland** [MJholland/navigation-highlight#1]
@thomasjbradley I finished this during class but markbot hangs while checking validations so I haven't been able to hand it in. 

![screen shot 2016-02-25 at 2 32 22 pm](https://cloud.githubusercontent.com/assets/16650658/13331770/107f06b6-dbcd-11e5-8dec-3605564734ef.png)

**Comment by thomasjbradley** [comment]
Hmm... have you tried quitting it and starting it again?
Maybe restarting your computer?

**Comment by MJholland** [comment]
Yeah, I tried both of those. I'm not sure if it's related to the javascript error that was coming up when Markbot opened.

**Comment by thomasjbradley** [comment]
I'm just sent out a new version of Markbot, maybe it will fix the error.
Check your Canvas inbox.

Let me know what happens.

**Comment by MJholland** [comment]
It's running fine now, thanks!
